lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -31,6 +31,10 @@
 
 begin
   require 'puppet-strings/tasks'
+  # strings:validate:reference would raise an error if REFERENCE.md is not
+  # up-to-date, but we only want to be sure that all documentation is in place
+  # and ensure it is kept up-to-date.
+  Rake::Task[:validate].enhance ['strings:generate:reference']
```

Comment:
this ahould only be run if the file exists. theREFERENCE.md is unuseable if params are documented in another way. at voy pupuli we fixed it like this: https://github.com/voxpupuli/voxpupuli-release/blob/master/lib/voxpupuli/release/rake_tasks.rb#L18

Reply:
Make sense, I updated the PR accordingly.